src/core/components/Content.tsx

```diff
@@ -79,24 +53,25 @@ export const ContentComponent = ({
     return () => {
       cancel = true;
     };
-  }, [content, modelConfig, hasError, onRendered]);
+  }, [content, modelConfig]);
+
+  useEffect(() => {
+    if (onRendered) {
+      requestAnimationFrame(onRendered);
+    }
+  }, [sanitizedContent, hasError, onRendered]);
 
   if (hasError) {
     return <span>Error: Could not render content!</span>;
   }
 
-  const hasThinkingContent = content.includes('<think>');
-
   return (
     <div data-testid="conversation-message" data-chat-role={role}>
-      {role === 'assistant' && hasThinkingContent && (
-        <ThinkingContent content={content} isStreaming={isRequesting} />
-      )}
       {sanitizedContent !== '' && (
         <span
           className={styles.content}
           dangerouslySetInnerHTML={{ __html: sanitizedContent }}
-        />
+        ></span>
```

Comment:
Don't like the single closed tag? lol

Reply:
i do, just copied the old Content component b4 recent changes.